operator/src/main/java/org/bf2/operator/operands/KafkaInstanceConfiguration.java

```diff
@@ -60,6 +60,10 @@
         @JsonProperty("min-margin")
         protected Quantity minMargin;
 
+        @JsonProperty("formatting-overhead")
+        protected double formattingOverhead = 105d/100;
```

Comment:
The default value of `105d/100` here is confusing to me. It evaluates to `1.05`, whereas we override in the properties file to `0.05`. Am I reading it correctly? What's the reason for the big difference?

Reply:
that's a mistake left from a previous version.  My intent is to have no default here and rely on the default from managedkafka.properties.

Reply:
fixed.